components/common/Link.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ Used for styling the links in the navbar
+ Will display children
+*/
+
+import { Link as ChakraLink } from "@chakra-ui/react";
+import { css } from "@emotion/css";
+
+const Link = (props: any) => {
```

Comment:
Did you try extending the existing Chakra UI `Link` component? 
[Here are the docs. ](https://chakra-ui.com/docs/styled-system/theming/component-style)
[Here's an example of the Button component. ](https://github.com/abhagsain/shardeum-website/blob/chakra-ui/theme/components/Button.ts)


Here's the folder structure [Chakra recommends](https://chakra-ui.com/docs/styled-system/theming/customize-theme#scaling-out-your-project)

I think we should try to extend as much as possible. Whatever Chakra currently has can be extended. 

This would give us better Typescript typings and we don't need to add unnecessary types for `children` as `React.ReactNode` etc which is also not present btw 😅

Reply:
Makes sense

Reply:
Done